core/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ generate_export_header(${TARGET_NAME})
 target_link_libraries(
   ${TARGET_NAME} PUBLIC scipp-common scipp-units Boost::boost
 )
-if(TBB_FOUND)
+if(TBB_FOUND AND NOT DISABLE_MULTI_THREADING)
```

Comment:
There are similar lines to this in `dataset/CMakeList.txt` and `variable/CMakeList.txt` do these need to be updated as well?

Reply:
Must have missed those, will take a look, but if we are not multithreading we do not need tbb.